### DISCUSSION TITLE
Add a shortcut to quickly enable or disable the video stream while leaving the audio stream and add sleep time

### DISCUSSION THREAD

**Discussion by trimechee** [smplayer-dev/smplayer#196] [Ideas]
Hello, please it will be amazing to add an option and shortcut il main toolbar to quickly enable or disable the video stream while leaving the audio stream because when we sleep, we can play a youtube video with SMPlayer but we just want to listen to the video without the video being displayed so the computer screen goes off and there is a saving of resources, and it will be great to add a sleep time mode to put the computer to sleep after a specific time ! 
Thank you.

**Comment by SketchMan3** [comment]
How do you even disable the video stream?

oh, apparently you disable video by putting in mplayer/mpv options:
--video=no

**Comment by trimechee** [comment]
Great ! Thank you :)
